src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -47,7 +47,14 @@ public async Task Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvid
                     filter(swagger, httpContext.Request);
                 }
 
-                await RespondWithSwaggerJson(httpContext.Response, swagger);
+                if (Path.GetExtension(httpContext.Request.Path.Value) == "yaml")
```

Comment:
Should probably support either .yml OR .yaml file extensions.

Reply:
Meh - maybe ignore that suggestion, given the following line from the Swagger docs, I think leaving it as is makes sense:

> It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.